src/components/RocketsItem.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import '../modules/RocketsItem.css';
+import { reserveRocket } from '../redux/Rockets/RocketsSlice';
+// import { reserveRocket } from '../redux/Rockets/RocketsSlice';
+
```

Comment:
- [ ] Same here, Please remove any commented code from your code base because it's considered good practice to do that.